src/Elements/Serialization/IFC/IFCModelExtensions.cs

```diff
@@ -157,7 +157,6 @@ internal static void ToIFC(Model model, string path)
             foreach (var m in model.AllElementsOfType<Material>())
             {
                 var material = new IfcMaterial(m.Name, null, "Hypar");
-                materials.Add(m.Name, material);
                 ifc.AddEntity(material);
```

Comment:
Will it cause any downstream issues to have two (possibly different) IFC materials with the same name?

Reply:
I think it comes down to how the application that consumes the IFC treats materials with the same name. There is nothing in the IFC spec that prevents this as `label`, the equivalent IFC field is optional.